packages/grid/helm/syft/templates/backend-service-account.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: backend-service-account
-  namespace: {{ .Release.Namespace }}
```

Comment:
Q: Why did we remove this?

Reply:
redundant. the helm deploys all the resources it in the namespace we provide as args. additionally, this was the only resource which had this value set, so just cleaning it up.